yellowbrick/classifier/prcurve.py

```diff
@@ -187,6 +196,8 @@ def __init__(
         model,
         ax=None,
         classes=None,
+        color=None,
```

Comment:
Do you mind if we make this parameter `colors` instead of `color`? Our API is unfortunately a little inconsistent in terms of how we name the colormap parameter (we originally used `cmap` but have been moving toward `colormap`), but I like the idea of converging on `colors` when the user is going to pass in a list of colors.

Reply:
`ClassificationScoreVisualizer` has `colors` property ([link](https://github.com/DistrictDataLabs/yellowbrick/blob/8689a2e9b04a4d96f476a3230431d3d445bc4cff/yellowbrick/classifier/base.py#L135)) and `set_params` ([link](https://github.com/DistrictDataLabs/yellowbrick/blob/8689a2e9b04a4d96f476a3230431d3d445bc4cff/yellowbrick/classifier/prcurve.py#L214)) raises an exception if I try to set `colors` parameter. I changed it to `color_names` to sound more specific, let me know if that works.